app/models/vm_or_template/right_sizing.rb

```diff
@@ -38,12 +38,12 @@ module VmOrTemplate::RightSizing
   module ClassMethods
     DEFAULT_CPU_RECOMMENDATION_MINIMUM = 1
     def cpu_recommendation_minimum
-      VMDB::Config.new("vmdb").config.fetch_path(:recommendations, :cpu_minimum) || DEFAULT_CPU_RECOMMENDATION_MINIMUM
+      Settings.recommendations.try(:cpu_minimum) || DEFAULT_CPU_RECOMMENDATION_MINIMUM
```

Comment:
We are moving all default values from code to yml.
Adding 
```
:recommendations:
  :cpu_minimum: 1
```
to ```config/settings.yml``` will allow call ```::Settings.recommendations.cpu_minimum```